rpc-server/src/modules/queries/methods.rs

```diff
@@ -116,13 +116,18 @@ async fn query_call(
             )
         }
 
-        let read_rpc_response_json = match &result {
-            Ok(res) => serde_json::to_value(res),
-            Err(err) => serde_json::to_value(err),
+        let (read_rpc_response_json, response_success) = match &result {
```

Comment:
I don't feel `response_success` is a good name for this variable. Maybe something like `is_response_ok` (`is_` could help identify it's a `bool`, `_ok` is closer to what we deal with). What do you guys think @kobayurii @telezhnaya ?

Reply:
100% agree